src/main/kotlin/net/minecrell/pluginyml/PlatformPlugin.kt

```diff
@@ -56,10 +58,8 @@ abstract class PlatformPlugin<T : PluginDescription>(private val platformName: S
                 }
             }
 
-            plugins.withType<JavaPlugin> {
-                tasks.named<AbstractCopyTask>("processResources") {
-                    from(generateTask)
-                }
+            project.extensions.getByType<SourceSetContainer>().named(SourceSet.MAIN_SOURCE_SET_NAME) {
+                resources.srcDir(generateTask.map { it.outputs })
```

Comment:
Mainly curious, have you tried if

```suggestion
                resources.srcDir(generateTask)
```

works?

Reply:
Tested just now and it seems like it does work